geps/gep-1897/index.md

```diff
@@ -149,61 +117,53 @@ certificate, we implement a metaresource named `BackendTLSPolicy`, that was prev
-by the application developer Gateway API persona to signal what the application developer _expects_ in connections to
-the application, from a TLS perspective.  Only the application developer can know what the application expects, so it is
-important that this configuration be managed by that persona.
-
-During the course of discussion of this proposal, we did consider allowing the cluster operator persona to have some access
-to Gateway cert validation, but as mentioned, BackendTLSPolicy is used primarily to signal what the application
-developer expects in the connection.  Granting this expectation to any other role would blur the lines between role
-responsibilities, which compromises the role-oriented design principle of Gateway API. As mentioned in Non-goal #8,
-providing a mechanism for the cluster operator gateway role to override gateway to backend TLS settings is not covered
-by this proposal, but should be addressed in a future update.  One idea is to use two types: ApplicationBackendTLSPolicy,
-and GatewayBackendTLSPolicy, where the application developer is responsible for the former, the cluster operator is
-responsible for the latter, and the cluster operator may configure whether certain settings may be overridden by
-application developers.
-
-The BackendTLSPolicy must contain these configuration items to allow the Gateway to operate successfully
+A new [metaresource](../gep-713/index.md), `BackendTLSPolicy`, will be introduced.
+
+With TLS, there are, of course, two components involved: the client (the gateway implementation) and the server (an application, likely running in the cluster).
+The Gateway API cannot control the server behavior.
+This is a choice the application developer makes in their code, whether or not (and how) they serve TLS.
+The application developer may not even be aware of the Gateway API at all.
+
+Therefor, the goal of this API is to tell the client *whether or not* and *how* to originate TLS to the backend.
+
+However, in many common cases these roles are actually inverted.
+While it is ultimately the client that must decide how to connect, commonly they are unaware of the details of the destination.
+In these situations, an application developer may wish to actually tell the clients how *they* should connect.
+
+Fortunately, these two scenarios can co-exist.
+`BackendTLSPolicy` will allow an application developer to present a *default* for clients that have not configured their own, but also allow clients to explicitly configure their own settings.
+
+This serves a variety of use cases:
+* A service producer may be unaware or unwilling to provide a `BackendTLSPolicy` for clients, despite serving TLS. In this case, a client can provide their own settings to connect
+  * Alternatively, a service producer may be unsure of how clients are connecting to them. `BackendTLSPolicy` implies *originating* TLS from the implementation, but the implementation could already be handling passthrough TLS.
```

Comment:
It makes sense to me that the implementation should recognize when passthrough is configured, and skip the BackendTLSPolicy in that case.

Reply:
Avoiding this was intentional. Having a security policy that is bypassed because the implementation attempts to be smart is a very tricky scenario. This opens the door to easily misconfigured security policies, and also breaks actual use cases where a user _does_ want the policy to apply.